genai-perf/genai_perf/config/generate/perf_analyzer_config.py

```diff
@@ -336,16 +342,6 @@ def _add_inference_load_args(self, config: ConfigCommand) -> List[str]:
 
         return inference_load_args
 
-    def _add_prompt_source_args(self, config: ConfigCommand) -> List[str]:
-        prompt_source_args = []
-        if (
-            config.input.prompt_source == PromptSource.PAYLOAD
-            and not "session_concurrency" in config.perf_analyzer.stimulus
-        ):
-            prompt_source_args += ["--fixed-schedule"]
-
-        return prompt_source_args
-
```

Comment:
As mentioned previously, GAP no longer infers when to send `--fixed-schedule` to PA, and does it if GAP user uses the new `--fixed-schedule` GAP option.

https://github.com/triton-inference-server/perf_analyzer/pull/377/files#r2087436633